v1_20/src/main/java/com/denizenscript/denizen/nms/v1_20/helpers/EntityHelperImpl.java

```diff
@@ -856,4 +858,10 @@ public void startUsingItem(LivingEntity entity, EquipmentSlot hand) {
     public void stopUsingItem(LivingEntity entity) {
         ((CraftLivingEntity) entity).getHandle().stopUsingItem();
     }
+
+    @Override
+    public void openHorseInventory(PlayerTag player, EntityTag horse) {
+        Horse craftHorse = ((CraftHorse) horse.entity).getHandle();

```

Comment:
Shouldn't this be casting to `CraftAbstractHorse`? also the naming convention is generally `nmsX`, I.e. `nmsHorse` or something in this case